couler/argo.py

```diff
@@ -242,10 +242,35 @@ def create_s3_artifact(
     )
 
 
-def create_secret(secret_data, namespace="default", name=None, dry_run=False):
+def obtain_secret(secret_keys, namespace="default", name=None, dry_run=False):
```

Comment:
Why not get_secret?

Reply:
Because sanity checks fail otherwise. Since there already is a function with that name in the script in [Line](https://github.com/couler-proj/couler/blob/master/couler/argo.py#L35)

Reply:
Then this function does not seem necessary? Users can just use `create_secret(use_existing=True)` directly?

Reply:
They could, but I understood from your previous reviews that you wanted this feature to be a separate function from the create_secret() but still use its result.